src/Link/Link.tsx

```diff
@@ -6,13 +6,16 @@ import { focusOutline } from '../utils/focusOutline'
 import { theme } from '../theme'
 import { Icon } from '../Icon'
 
+type LinkTypo = 'regular' | 'small'
+
 export type LinkProps = {
   className?: string
   href: string
   onClick?: MouseEventHandler
   openInNewTab?: boolean
   children?: ReactNode
   download?: boolean
+  typo?: LinkTypo
```

Comment:
I don't think we want to make it customisable, it should inherit the styling from its parent instead?

Reply:
typography has 2 new styles for the link and this will never be wrapped by a `Text` component so we need them here

Reply:
Looking at Figma, it seems to match with Text typo `body-regular` and `body-small`

I don't see any reason to be able to customise the `Link` size, except inside a paragraph so I'm guessing that the size `small` for `Link` should only be used when inside `body-small` and the default and the one that should be used in most situation is `regular`

Could we confirm with design?

Reply:
ok, let me check with Alistair. I still think that we need those sizes here since if we use the link outside of a paragraph we don't have any option to set the font and I don't think it should be wrapped by a `Text` component just to get the typo

Reply:
Yeah definitely, my comment was with the assumption that we never need Link `small`, except in `body-small`
And Link `regular` should be used in most situation

If there is a need for Link `small` tho, I'm good with adding a prop for it

Reply:
@mickaelzhang I've checked with Alistair and we agreed that might be safer to keep the style as a prop cause for now there's not a case where we use links outside paragraphs but we can't 100% be sure that in the future there won't be.
So I'd rather keep the prop now and have to always add it if we need a small Link then to have to go back and change it across all repos in future

Reply:
If we don't have a case for having it outside of paragraph for now, wouldn't it be safer to not have a prop?
Adding a prop is easy compared to remove it and check if there hasn't been bad usage across app?

Reply:
It's currently defaulting to `regular` anyway so we'll only need to add the `small` in case we need it and I'd rather have the prop from the start in case we start using standalone links outside paragraphs in the future rather than having to wrap them in a `Text` to inherit the style